### ISSUE TITLE
Demystifying documentation

### ISSUE THREAD

**Issue by cararthompson** [rfortherestofus/marketing#9]
This sounds really silly but look me a while at the start of using R to not be daunted by documentation and stackoverflow. What is this magic "foo" "bar" incantation? Do I leave the <>s around the word I know I need to replace? We could write something to demystify common things that get used by programmers and take a bit of getting used to. It ties in with consulting in terms of how we make our documentation easy for clients to read.

**Comment by dgkeyes** [comment]
No, this is not silly at all! This is EXACTLY what newbies need. @charliejhadley and I were just talking about this.

**Comment by cararthompson** [comment]
Great! If we can think of more examples, these could easily be a series of tweets. Should we add more within this issue or create a new issue for each?

**Comment by dgkeyes** [comment]
We also created a thread for R in 3 Months here: https://rfortherestofus.com/groups/r-in-3-months-fall-2021/forum/discussion/how-to-read-r-docs-pages/

**Comment by cararthompson** [comment]
Looking forward to working on this one. Can you grant me access to the url above? Thanks!

**Comment by dgkeyes** [comment]
Just created an account on the R for the Rest of Us website for you @cararthompson. You should receive an email at your R for the Rest of Us email. Once you get that, sign in to the website and you should be able to access that URL.
